doc/configuration.md

```diff
@@ -77,13 +80,13 @@ notifications:
 
 N.B. In some instances a configuration option may be marked **optional** but contain child options marked as **required**. This indicates that a parent may be omitted with all its children, however, if the parent is included, the children marked **required** must be included.
 
-## version 
+## version
 
 ```yaml
 version: 0.1
 ```
 
-The version option is **required** and indicates the version of the configuration being used. It is expected to remain a top-level field, to allow for a consistent version check before parsing the remainder of the configuration file. 
```

Comment:
I know it must be obnoxious to run into formatting issues, but please avoid reformatting in unrelated PRs. We are willing to accept a separate PR to fix the document formatting.